node/utils.go

```diff
@@ -310,13 +311,22 @@ func createVolumeDir(hostPath, containerSpec, imageSpec, userSpec, permissionSpe
 	createVolumeDirMutex.Lock()
```

Comment:
thank you - i was also thinking along your lines - hostpath based map of locks - but deferred to the simpler/easier solutions